packages/react-sdk/README.md

```diff
@@ -0,0 +1,91 @@
+# Bucket React SDK
+
+React client side library for [Bucket.co](https://bucket.co)
+
+## Install
+
+Install via npm:
+
+```
+npm i @bucketco/react-sdk
+```
+
+## Setup
+
+Wrap your application with the `Bucket` higher order component.
+
+This will initialize Bucket, fetch feature flags and start listening for Live Satisfaction events.
+
+```tsx
+<Bucket publishableKey="{YOUR_PUBLISHABLE_KEY}">{/* ... */}</Bucket>
+```
+
+### Props
+
+All options which can be passed to `bucket.init` can be passed as props to the Bucket higher order component.
+
+```tsx
+<Bucket
+  publishableKey="{YOUR_PUBLISHABLE_KEY}" // The publishable key of your app environment
+  debug={false} // Enable debug mode to log info and errors
+  persistUser={true} // See the Tracking SDK documentation under "Persisting Users"
+  host="https://tracking.bucket.co" // Configure the host Bucket calls are made to
+  sseHost="https://livemessaging.bucket.co" // Configure the host Bucket sse calls are made to
+  feedback={{
+    // See feedback options here: https://github.com/bucketco/bucket-tracking-sdk/blob/main/packages/tracking-sdk/FEEDBACK.md#global-feedback-configuration
+  }}
+>
+```
+
+## Hooks
```

Comment:
hey @Munter @Swiftwork . Would be great to get your input on this API. I know you had some thoughts.